### ISSUE TITLE
Quick guide page not found

### ISSUE THREAD

**Issue by realdreamer** [plepers/nanogl#5]
Quick guide links are broken in API documentation page. When try to open page not found error.

**Comment by impressivewebs** [comment]
Yeah, I noticed this too. For those looking for the docs, they are linked correctly in the GitHub readme:

https://github.com/plepers/nanogl/blob/develop/README.md
